openqa-qam-sle

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    cat << EOF
+Usage: $(basename "$0") -a SUSE:Maintenance:II:RR [-d days]
+
+Search openQA jobs related to a given update inside job groups:
+- Maintenance: Single Incidents / Core Incidents
+- Maintenance: Aggregated updates / Core Maintenance Updates
+Default search within the last 5 days for "Aggregated updates" builds
+
+Usage: eg
+  openqa-qam-sle -a SUSE:Maintenance:34576:337335
+  openqa-qam-sle -a SUSE:Maintenance:34577:337331
+  openqa-qam-sle -a SUSE:Maintenance:34564:337279
+  openqa-qam-sle -a SUSE:Maintenance:34581:337626
+EOF
+}
+
+# Maintenance: Single Incidents / Core Incidents
+declare -A dict_group=(
+    ["15-SP1"]=233
+    ["15-SP2"]=306
+    ["15-SP3"]=367
+    ["15-SP4"]=439
+    ["15-SP5"]=490
+    ["15-SP6"]=546
+    ["12-SP3"]=106
+    ["12-SP5"]=282
+    ["15-SP4-TERADATA"]=521
+    ["12-SP3-TERADATA"]=191
+)
+
+readonly url_dashboard_qam='http://dashboard.qam.suse.de'
+readonly url_openqa='https://openqa.suse.de'
+readonly url_qam='https://qam.suse.de'
+
+_ok() {
+    echo -e "\033[01;32m$1\033[0m"
+}
+
+_ko() {
+    echo -e "\033[01;31m$1\033[0m"
+}
+
+_warn() {
+    echo -e "\033[01;33m$1\033[0m"
+}
+
+_title() {
+    echo -e "\033[01;36m$1\033[0m"
+}
+
+search_maintenance_single_incidents() {
+    _title "Maintenance: Single Incidents / Core Incidents"
+    local review_request_id=$1
+
+    local ID
+    ID="$(echo "$review_request_id" | awk -F':' '{printf "%s", $3}')" # The ID of the update
+
+    local incident_settings
+    incident_settings=$(curl -sk -X GET "${url_dashboard_qam}/api/incident_settings/$ID")
+
+    local build
+    build=$(echo "$incident_settings" | jq -r '.[0].settings.BUILD')
+
+    local versions
+    versions=$(echo "$incident_settings" | jq -r '.[].settings.VERSION' | sort | uniq)
+
+    local versions_teradata
+    versions_teradata=$(echo "$incident_settings" | jq -r '[.[]| { KEY:(.settings.VERSION + "|" +.settings.FLAVOR)}] | .[].KEY' | grep TERADATA | sed -e 's/|.*-TERADATA$/-TERADATA/' | sort | uniq)
+
```

Comment:
Please treat anything related to that term confidential

Reply:
I keep in mind

Reply:
Well, `teradata` isn't necessarily confidential: https://github.com/search?q=repo%3Aos-autoinst%2Fos-autoinst-distri-opensuse%20teradata&type=code

Reply:
yeah good point. I would still prefer to avoid it here.

Reply:
ok, nevermind. We have https://www.suse.com/de-de/success/teradata/ :)